docs/swupd.1.rst

```diff
@@ -125,6 +125,10 @@ used to modify the core behavior and resources that swupd uses.
 
    Set the maximum number of parallel downloads
 
+- ``--no-progress``
+
+   Don't report progress
```

Comment:
I think this should say something a little more explanatory for example "Don't report **download** progress", because just saying "Don't report progress" I think can be confused with running a command "quietly", the output from a command is still some kind of progress report.